### PR TITLE
DAOS-4854 control: update ras event naming and severity levels

### PR DESCRIPTION
**Pull request by tanabarr**
* update event names (DAOS-7262)
* update severity levels (DAOS-7263)
* change rank_down to engine_died (DAOS-7265)
* add engine format required event (DAOS-6889)

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

### CODE REVIEW

**Comment by tanabarr** [comment]
Gatekeeper: please use PR title and description as commit message when landing, thanks in advance

File: src/control/server/mgmt_system.go [hunk 1]
```diff
@@ -613,15 +612,19 @@ func (svc *mgmtSvc) SystemStop(ctx context.Context, pbReq *mgmtpb.SystemStopReq)
...
-		}))
-	}
+	// if pbReq.GetHosts() == "" && pbReq.GetRanks() == "" && pbReq.GetKill() {
+	// 	svc.events.Publish(events.New(&events.RASEvent{
+	// 		ID:   events.RASSystemStop,
+	// 		Type: events.RASTypeInfoOnly,
+	// 		Msg:  "System-wide shutdown requested",
+	// 		Rank: uint32(system.NilRank),
+	// 	}))
+	// }
```
Comment (kjacque) [inline src/control/server/mgmt_system.go:627]: Looks like perhaps this and the system start requested event below were intended to be fully removed?

**Review by kjacque — APPROVED** [review]

File: src/control/server/mgmt_system.go (hunk 1 above)
Comment (tanabarr) [reply src/control/server/mgmt_system.go:627]: the commented blocks are proceeded with a ticket number, it's a placeholder for modifying the events as the referenced ticket describes, the code had to be commented as this PR changes the event IDs and makes the comment to code invalid, the ticket for updating the system start/stop events is to be done imminently

Comment (kjacque) [reply src/control/server/mgmt_system.go:627]: I saw the ticket number, it just looked like the planned new events are totally different from the old ones being removed, so it wasn't clear that the old lines would make a good reference. Not that important in any case.

**Review by mjmac — APPROVED** [review]

**Pull request merged**